pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -986,7 +1126,7 @@ public PinotDataType getSingleValueType() {
   /**
    * Returns the {@link PinotDataType} for the given {@link FieldSpec} for data ingestion purpose. Returns object array
    * type for multi-valued types.
-   * TODO: Add MV support for BOOLEAN, TIMESTAMP, BYTES
+   * TODO: Add MV support for BOOLEAN, TIMESTAMP, BYTES, JSON
```

Comment:
I don't think we need MV support for JSON as it can hold array

Reply:
Fixed.